test/utils/utils.go

```diff
@@ -5,9 +5,11 @@
 package utils
 
 import (
+	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
 )
 
 func ExpectNoError(err error, extra ...interface{}) {
+	defer ginkgo.GinkgoRecover()
```

Comment:
why is this needed?
isn't this already done by the fail handler registration in the test.